app/controllers/tenants/projects_controller.rb

```diff
@@ -6,7 +6,9 @@ def index
     redirect_to root_path if @tenant.nil?
 
     if params[:category_name] == "Shop All" || params[:category_name].nil?
-      @projects = Project.includes(:photos).active
+      @projects = Project.paginate(page: params[:page],
+                                  per_page: 10).
```

Comment:
somebody please take a look at this.  This is showing up as correct in my editor but I'm not sure until somebody look at it in theres